Cannot compile

running `cargo build` returns
```
   Compiling cqlite v0.2.2 (/Users/cscetbon/src/git/cqlite)
error[E0004]: non-exhaustive patterns: `sanakirja::Error::Corrupt(_)` not covered
   --> src/error.rs:77:15
    |
77  |         match error {
    |               ^^^^^ pattern `sanakirja::Error::Corrupt(_)` not covered
    |
note: `sanakirja::Error` defined here
   --> .cargo/registry/src/github.com-1ecc6299db9ec823/sanakirja-1.2.16/src/lib.rs:133:5
    |
118 | pub enum Error {
    | --------------
...
133 |     Corrupt(u64),
    |     ^^^^^^^ not covered
    = note: the matched value is of type `sanakirja::Error`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern or an explicit pattern as shown
    |
80  ~             SanakirjaError::VersionMismatch | SanakirjaError::CRC(_) => Self::Corruption,
81  ~             sanakirja::Error::Corrupt(_) => todo!(),
    |

For more information about this error, try `rustc --explain E0004`.
error: could not compile `cqlite` due to previous error
```

--- a/src/error.rs
+++ b/src/error.rs
@@ -78,6 +78,7 @@ impl From<SanakirjaError> for Error {
             SanakirjaError::IO(err) => Self::IO(err),
             SanakirjaError::Poison => Self::Poison,
             SanakirjaError::VersionMismatch | SanakirjaError::CRC(_) => Self::Corruption,
+            SanakirjaError::Corrupt(_) => Self::Corruption,
         }
     }
 }

--- a/src/store/txn.rs
+++ b/src/store/txn.rs
@@ -1,5 +1,5 @@
 use crate::Error;
-use sanakirja::{AllocPage, Commit, Env, LoadPage, MutTxn, RootDb, RootPage, Txn};
+use sanakirja::{AllocPage, Commit, Env, LoadPage, MutTxn, RootDb, RootPage, Txn, MutPage};
 use std::borrow::Borrow;
 
 pub(crate) enum DynTxn<E: Borrow<Env>> {
@@ -103,6 +103,13 @@ impl<E: Borrow<Env>> AllocPage for DynTxn<E> {
         match self {
             DynTxn::Txn(_) => Err(Error::ReadOnlyWrite),
             DynTxn::MutTxn(txn) => Ok(txn.decr_rc_owned(off)?),
+        }
+    }
+
+    fn alloc_contiguous(&mut self, off: u64) -> Result<MutPage, <Self as LoadPage>::Error> {
+        match self {
+            DynTxn::Txn(_) => Err(Error::ReadOnlyWrite),
+            DynTxn::MutTxn(txn) => Ok(txn.alloc_contiguous(off)?),
         }
     }
 }

